README.md

```diff
@@ -105,6 +105,14 @@ Removed items expire no less than `revisionRetentionPolicy.grace_ttl` seconds.
 When `revisionRetentionPolicy.type` is `ttl`, all items are maintained no less than for
 `revisionRetentionPolicy.ttl` seconds, and expire after that period.
 
+## Custom TTL
+A custom TTL could be set for a individual object on `PUT` requests by providing a special
+`_ttl` integer attribute. It's value is the number of seconds an object would exist in the
+database after insertion.
+
+Please note, that setting custom `_ttl` for individual rows in not allowed for tables 
```

Comment:
*... is not allowed for tables which have a revision retention policy set, as table-wide and row-wide TTLs would conflict.*